packages/bpk-mixins/src/mixins/_forms.scss

```diff
@@ -486,13 +486,13 @@ $bpk-spacing-v2: true;
 @mixin bpk-checkbox {
   position: relative;
   display: inline-block;
-  padding-left: bpk-spacing-sm() * 7;
+  padding-left: bpk-spacing-lg() + bpk-spacing-sm();
```

Comment:
Changed the tokens used to make it clearer for the spacing - the value is the exact same as before just using a different combinations of tokens to achieve the same